contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>EU hosting or data residency available</td>
+        </tr>
+        <tr>
+            <td><strong>HIPAA compliant</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Self-host only</td>
+            <td>Business Associate Agreement available</td>
+        </tr>
+        <tr>
+            <td><strong>Data anonymization</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Remove personally identifiable information</td>
+        </tr>
+        <tr>
+            <td><strong>Self-hosting option</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Self-host and keep complete control of your data</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+## Frequently asked questions
+
+- [How is PostHog different?](#how-is-posthog-different)
+  - [1. PostHog is an all-in one platform](#1-posthog-is-an-all-in-one-platform)
+  - [2. PostHog is built for engineers](#2-posthog-is-built-for-engineers)
+  - [3. PostHog is open source](#3-posthog-is-open-source)
+- [Core features](#core-features)
+  - [Session replays](#session-replays)
+  - [Heatmaps](#heatmaps)
+  - [In-app messages and user research](#in-app-messages-and-user-research)
+  - [Apps and integrations](#apps-and-integrations)
+- [Privacy, compliance, and security](#privacy-compliance-and-security)
+- [Frequently asked questions](#frequently-asked-questions)
+  - [Who is Hotjar useful for?](#who-is-hotjar-useful-for)
+  - [Who is PostHog useful for?](#who-is-posthog-useful-for)
+  - [How much does Hotjar cost?](#how-much-does-hotjar-cost)
+  - [How much does PostHog cost?](#how-much-does-posthog-cost)
+  - [Do Hotjar and PostHog offer free trials?](#do-hotjar-and-posthog-offer-free-trials)
+
+Got another question? You can [ask the PostHog team anything you want](/questions)!
+
+### Who is Hotjar useful for?
+
+Hotjar is designed primarily for marketers and UX specialists. Its key features are the ability to record session replays and heatmaps. It also provides widgets for collecting feedback and tools for scheduling customer interviews. These provide insights into how customers are using your product. 
+
+### Who is PostHog useful for?
+
+PostHog is built primarily with engineers, product managers and other technical users in mind. It also offers the ability to record session replays, heatmaps, and create feedback widgets, but also offers a number of other tools that are useful for these types of teams, such as product analytics and feature flags. 
```

Comment:
Generally, I think we should be throwing links to relevant docs and product pages in here.